### ISSUE TITLE
Cannot install from npm

### ISSUE THREAD

**Issue by kirilloid** [jviereck/regjstraverse#2]
`npm` tries to `chmod` `node_modules/regjstraverse/bin/parser` and fails because there's no such file.

So I think it doesn't really matter, but still:
> Windows_NT 10.0.14393
> node v6.9.2
> npm  v3.10.9

**Comment by jviereck** [comment]
Thanks a lot for reporting this @kirilloid! This should be fixed with the release 0.1.2. It got published to npm and I was able to install the module from there now:

```bash
➜  tmp npm install regjstraverse
/Users/jviereck/dev/tmp
└── regjstraverse@0.1.2
```

@kirilloid: Could you maybe let me know if this works for you now?

**Comment by kirilloid** [comment]
Thanks for prompt reaction.
